src/SenseoWifi.cpp

```diff
@@ -251,14 +253,29 @@ void senseoStateEntryAction() {
   }
 }
 
+void onHomieEvent(const HomieEvent &event)
+{
+  // The device rebooted when attachInterrupt was called in setup()
+  // before Wifi was connected and interrupts were already coming in.

```

Comment:
Is that so? Maybe new behavior with the latest code base I didn't catch. Thanks for the additional fix ;)
In case you add another commit to this PR: Could you please format this part in alignment with the code style of the file? Thanks!

Reply:
The interrupts begin to fire before wifi is up. When the led-change event is triggered but the wifi connection is not established it may results in an error accessing the broker.